recipes/consensusfixer/ConsensusFixer.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# ConsensusFixer wrapper script
+# based on picard wrapper
+set -eu -o pipefail
+
+set -o pipefail
+export LC_ALL=en_US.UTF-8
+
+# Find original directory of bash script, resovling symlinks
+# http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in/246128#246128
+# this is basically a bash re-implementation of the (GNU-only, non Mac OS X) "realpath" command
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
```

Comment:
double brackets `[[ ]]`, please.

Reply:
I think my first attempt at answer got lost somewhere...

 - `test` and `[` bash built-ins clone the behaviour of the POSIX `test` command line tool.
 - `[[` built-in is a bash specific extension that adds C-style logical operation, pattern matching, regex matching, etc.

Here the script's original author uses standard behaviour (checking for symlinks) so the POSIX-compliant bash built-in suffices.

Reply:
I am well aware of the difference between POSIX `[` and the bashism `[[`. There are good reasons to prefer `[[` over POSIX-isms, one of which is the fact that `[[` provides for less surprising behaviour than the POSIX stuff. https://stackoverflow.com/a/669486
Gentoo also recommends it over POSIX shell features, https://devmanual.gentoo.org/tools-reference/bash/index.html#single-versus-double-brackets-in-bash
Please just do this.

Reply:
And none of the weird surprising edge case are hit by the above code (which usually have to do with file pattern matching,  badly quoted empty string matching, etc.)
But, well, it's your call. Let's get the thing merged already... 
(squashed/rebased/pushed, waiting for Travis)